### PR TITLE
APPDUX-81: Add role info under user menu

### PR DESCRIPTION
**Pull request by mfrances17**
## Motivation
https://issues.redhat.com/browse/APPDUX-81

## What
Add login name, admin/dev icon, and admin/dev 'badge' based on whether the current user has admin rights or not, according to the design spec.

## Why
To better display whether the currently logged in user has admin account privileges or not.

## Verification Steps
No cluster to test on, so you will need to verify by running locally:
1. Start solution explorer and click the user menu in the masthead. It should display the administrator version of the menu.
2. Change ln 230 of masthead.js to 'isAdmin = false'.
3. Click the user menu in the masthead. It should now display the developer version of the menu.

## Checklist:
- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

## Progress
- [x] Finished task

## Additional Notes
**User menu when logged in as Admin:**
admin_user_menu

**User menu when logged in as Dev:**
dev_user_menu

### CODE REVIEW

**Comment by tiffanynolan** [comment]
Looking good! Just a few styling tweaks. 

- No hover state/background color change for the profile info section. 
- Background color for the section with the profile info: global_BackgroundColor_150
- Background color for the developer label should be: global_palette_cyan_100

**Comment by mfrances17** [comment]
> Looking good! Just a few styling tweaks.
> 
> * No hover state/background color change for the profile info section.
> * Background color for the section with the profile info: global_BackgroundColor_150
> * Background color for the developer label should be: global_palette_cyan_100

Thanks @tiffanynolan... I made the CSS changes. I've attached the latest screen caps, and/or you can run it locally and let me know if you approve.

admin

dev

File: src/styles/application/components/_masthead.scss [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
   color: var(--pf-global--disabled-color--100);
 }
+
+.user-menu > div {
+  padding: 2px;
+}
+
+.user-menu {
+  display: grid;
+  grid-template-columns: minmax(75px, auto) 1fr;
```
Comment (christiemolloy) [inline src/styles/application/components/_masthead.scss:13]: nice work using grid 😀

**Comment by mfrances17** [comment]
> Looks great! Thanks, Mark.

Thanks... here's a screenie of the correct dev color, just to keep me honest.

correct-dev-color

**Pull request merged**